### DISCUSSION TITLE
building-the-minimal-asp-net-core-app-with-webpack-and-npm/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#53] [Blog comments]
# building-the-minimal-asp-net-core-app-with-webpack-and-npm/

https://blogs.taiga.nl/martijn/posts/building-the-minimal-asp-net-core-app-with-webpack-and-npm/

**Comment by disqus-comments-migrator** [comment]
<em>[Mudassar Rauf](https://disqus.com/by/mudassarrauf/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 03 2017, 04:46</em>

---

<p>A very nice approach. I was looking for one like this. <br>The same thing could be applied to all the built in templates.</p>

**Comment by disqus-comments-migrator** [comment]
<em>[Brendon](https://disqus.com/by/brendonmiszka/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 21 2017, 06:06</em>

---

<p>Great article!</p><p>1. For BootStrap 4 (beta 2) I had an issue with it not finding Popper. If anyone has this problem, see <a href="https://getbootstrap.com/docs/4.0/getting-started/webpack/" rel="nofollow noopener" title="https://getbootstrap.com/docs/4.0/getting-started/webpack/">https://getbootstrap.com/docs/4.0/getting-started/webpack/</a> and the plugin section to add to your webpack.config.js file.<br>2. This all seems to work just about the same if using Yarn instead of npm. This includes the linked Visual Studio task runner extension - luckily that also supports Yarn.</p>

**Reply by martijnboland** [reply]
<em>[Martijn Boland](https://disqus.com/by/martijnboland/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 21 2017, 09:15, in reply to [Brendon](https://disqus.com/by/brendonmiszka/)'s [comment](https://github.com/martijnboland/blog/discussions/53#discussioncomment-8952989)</em>

---

<p>Thanks for the tips, much appreciated!</p>

**Reply by disqus-comments-migrator** [reply]
<em>[Brendon](https://disqus.com/by/brendonmiszka/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 29 2017, 04:42, in reply to [Martijn Boland](https://disqus.com/by/martijnboland/)'s [comment](https://github.com/martijnboland/blog/discussions/53#discussioncomment-8952991)</em>

---

<p>Something else of interest. Had a strange issue where the dist folder wasn't being included in the publish the first time publish was run. Noticed on CI server. Replacing BeforeTargets="Publish" with BeforeTargets="PrepareForPublish" seemed to fix it. There may well be other solutions, based on a few Google searches including task runner generated files in the publish is a bit of a problem area.</p>

**Comment by disqus-comments-migrator** [comment]
<em>[sinand99](https://disqus.com/by/sinand99/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 24 2017, 01:22</em>

---

<p>Thanks for the article, but please update your minimal app for Asp.NET Core 2.0</p>

**Reply by martijnboland** [reply]
<em>[Martijn Boland](https://disqus.com/by/martijnboland/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 24 2017, 10:47, in reply to [sinand99](https://disqus.com/by/sinand99/)'s [comment](https://github.com/martijnboland/blog/discussions/53#discussioncomment-8952993)</em>

---

<p>Good point, will do</p>

**Reply by martijnboland** [reply]
<em>[Martijn Boland](https://disqus.com/by/martijnboland/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 29 2017, 09:27, in reply to [sinand99](https://disqus.com/by/sinand99/)'s [comment](https://github.com/martijnboland/blog/discussions/53#discussioncomment-8952993)</em>

---

<p>Updated post: <a href="https://blogs.taiga.nl/martijn/2017/11/24/building-and-asp-net-core-mvc-app-with-npm-and-webpack-asp-net-core-2-0-edition/" rel="nofollow noopener" title="https://blogs.taiga.nl/martijn/2017/11/24/building-and-asp-net-core-mvc-app-with-npm-and-webpack-asp-net-core-2-0-edition/">https://blogs.taiga.nl/martijn/2017/11/24/building-and-asp-net-core-mvc-app-with-npm-and-webpack-asp-net-core-2-0-edition/</a></p>

**Reply by disqus-comments-migrator** [reply]
<em>[sinand99](https://disqus.com/by/sinand99/) commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at november 29 2017, 05:47, in reply to [Martijn Boland](https://disqus.com/by/martijnboland/)'s [comment](https://github.com/martijnboland/blog/discussions/53#discussioncomment-8952997)</em>

---

<p>Thanks Martijn!</p>
